gui/mozregui/report.py

```diff
@@ -1,17 +1,24 @@
 from PySide6.QtCore import QAbstractTableModel, QModelIndex, Qt, QUrl, Signal, Slot
-from PySide6.QtGui import QColor, QDesktopServices
+from PySide6.QtGui import QColor, QDesktopServices, QGuiApplication, QPalette
 from PySide6.QtWidgets import QTableView, QTextBrowser
 
 from mozregression.bisector import NightlyHandler
 
 # Custom colors
 GRAY_WHITE = QColor(243, 243, 243)
+DARK_GRAY = QColor(28, 28, 28)
 VERDICT_TO_ROW_COLORS = {
     "g": QColor(152, 251, 152),  # light green
     "b": QColor(250, 113, 113),  # light red
     "s": QColor(253, 248, 107),  # light yellow
     "r": QColor(225, 225, 225),  # light gray
 }
+VERDICT_TO_ROW_COLORS_DARK_MODE = {
+    "g": QColor(48, 209, 88),  # green
+    "b": QColor(255, 70, 58),  # red
+    "s": QColor(160, 90, 0),  # yellow
+    "r": QColor(45, 45, 45),  # gray
+}
```

Comment:
I'm not sure these belong in a separate dictionary. They could potentially be added with different keys to `VERDICT_TO_ROW_COLORS`.